content/webapp/services/prismic/transformers/exhibitions.ts

```diff
@@ -13,17 +11,123 @@ import { PaginatedResults } from '@weco/common/services/prismic/types';
 import { transformQuery } from './paginated-results';
 import { london } from '@weco/common/utils/format-date';
 import { transformMultiContent } from './multi-content';
+import {
+  asText,
+  isDocumentLink,
+  isEmptyHtmlString,
+  parseBoolean,
+  parseGenericFields,
+  parseImagePromo,
+  parsePlace,
+  parsePromoToCaptionedImage,
+  parseSingleLevelGroup,
+  parseTimestamp,
+  parseTitle,
+} from '@weco/common/services/prismic/parsers';
+import { link } from './vendored-helpers';
+import { breakpoints } from '@weco/common/utils/breakpoints';
+import {
+  parseExhibitionFormat,
+  parseExhibits,
+  parseResourceTypeList,
+} from '@weco/common/services/prismic/exhibitions';
+import { parseSeason } from '@weco/common/services/prismic/seasons';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function transformExhibition(
   document: ExhibitionPrismicDocument
 ): Exhibition {
-  const exhibition: DeprecatedExhibition = parseExhibitionDoc(document);
+  const genericFields = parseGenericFields(document);
+  const data = document.data;
+  const promo = data.promo;
+  const exhibits = data.exhibits
+    ? data.exhibits.map(i => link(i.item) && i.item.id)
+    : [];
+  const events = data.events
+    ? data.events.map(i => link(i.item) && i.item.id)
+    : [];
+  const articles = data.articles
+    ? data.articles.map(i => link(i.item) && i.item.id)
+    : [];
+  const relatedIds = [...exhibits, ...events, ...articles].filter(Boolean);
```

Comment:
We also looked for `data.books` and `data.pages` in the JavaScript implementation; TS tells us this is unnecessary because you can't link to those types from an exhibition in Prismic.